Sources/Sentry/SentryDispatchQueueWrapper.m

```diff
@@ -3,13 +3,7 @@
 
 NS_ASSUME_NONNULL_BEGIN
 
-@implementation SentryDispatchQueueWrapper {
-    // Don't use a normal property because on RN a user got a warning "Property with 'retain (or
-    // strong)' attribute must be of object type". A dispatch queue is since iOS 6.0 an NSObject so
-    // it should work with strong, but nevertheless, we use an instance variable to fix this
-    // warning.
-    dispatch_queue_t queue;
-}
```

Comment:
This is no longer true since we bumped our min supported version. I believe we can also change this: https://github.com/getsentry/sentry-cocoa/blob/d257eb9dc69e0ffa45c1e4e526fa0f5d7199b83c/Sources/Sentry/SentryDispatchQueueWrapper.m#L16-L17

This is now exposed as a property in the public interface so it can be accessed when setting up a new `dispatch_source_t` in `SentryDispatchSourceWrapper`[